cabot/cabotapp/jenkins.py

```diff
@@ -1,6 +1,6 @@
 from datetime import datetime
 
-import requests
+from jenkinsapi.jenkins import Jenkins
 from celery.utils.log import get_task_logger
 from django.conf import settings
 from django.utils import timezone
```

Comment:
On line 10 (I can't comment on lines not in the diff? :/), `auth = (settings.JENKINS_USER, settings.JENKINS_PASS)`

auth is no longer used